### ISSUE TITLE
Webstorm UI integration doesn't show up

### ISSUE THREAD

**Issue by ThaNarie** [wallabyjs/quokka#43]
### Issue description or question

I have a hard time having the IDE integration to show.
The panel shows all the output, but the gutter indicators and inline output is not showing up.

I tried restarting from the panel, re-opening the file and starting it from the context menu, re-opening the project, etc.
It sometimes shows up, but couldn't find a reproducible case.

**Update**: seems like creating new scratch files starts Quokka automatically including the visual UI.
But when closing and re-opening those scratch files, and starting it form the context menu (or from existing js files), doesn't show the visual UI.

Having babel enabled or not doesn't make a difference.

### Quokka.js file

```javascript
function foo(a) {
  console.log('foo', a);
  a.toString();
  return a/*?*/ * 2;
}

console.log(foo(123));
```

### Quokka.js Console Output

```
Quokka #1 (node: v6.9.5, babel: v6.14.0)

foo 123 at test-foo.js:2:2

123 at a test-foo.js:4:2

246 at foo(123) test-foo.js:7:0
```

### Code editor version

* Quokka 1.0.3
* WebStorm 2017.1.2 (april 25 build)

### OS name and version

OSX 10.12.3

**Comment by ArtemGovorov** [comment]
Could you please re-start your, try running Quokka on an existing file and paste the recent output from the idea.log file (menu `Help - Show Log In Finder`)?

**Comment by blackdynamo** [comment]
I think this is fixed in 1.0.6

**Comment by ArtemGovorov** [comment]
@blackdynamo Awesome, thanks for the update!

**Comment by ThaNarie** [comment]
It seems to be fixed in 1.0.7! (1.0.6 still didn't look solid, but the 1.0.7 update was already out so I tried that).

**Comment by ArtemGovorov** [comment]
@ThaNarie Yep, we were able to reproduce few glitches, like those you've described, in 1.0.6 and fixed them in 1.0.7. Thanks for the update!
